README.md

```diff
@@ -1,65 +1,86 @@
-# Welcome to Streamlit :wave:
+![Streamlit logo|200x200](https://i.ibb.co/5FdSr1V/streamlit.png)
 
-**The fastest way to build and share data apps.**
+# Streamlit
 
-Streamlit lets you turn data scripts into shareable web apps in minutes, not weeks. It’s all Python, open-source, and free! And once you’ve created an app you can use our [Community Cloud platform](https://streamlit.io/cloud) to deploy, manage, and share your app!
+**A faster way to build and share data apps.**
+
+Streamlit lets you turn data scripts into shareable web apps in minutes, not weeks. It’s all Python, open-source, and free! And once you’ve created an app you can use our [Community Cloud platform](https://streamlit.io/cloud)  to deploy, manage, and share your app!
```

Comment:
two spaces after the cloud link ;)

Reply:
Well spotted! Remember you can use the "Add a suggestion" command ;-) 
<img width="177" alt="image" src="https://user-images.githubusercontent.com/7164864/215104490-54371472-1529-40bb-95a1-5c6627d7682c.png">